src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -28,6 +28,7 @@
       <VersionFeature21>30</VersionFeature21>
       <VersionFeature31>21</VersionFeature31>
       <VersionFeature50>12</VersionFeature50>
+      <VersionFeature60>0</VersionFeature60>
```

Comment:
This should be 1 now I think.

Reply:
@wtgodbe are there targeting packs now getting updated as well and should they use the same version? Is that true for all three runtimes?

Reply:
All 3 targeting packs are revving for 6.0.2, and all have packageVersion 6.0.2. They won't be available until Feb